app.use(function onerror(err, req, res, next) {
  // an error occurred!
});
